### PR TITLE
Don't change resharer when resharing/add mobile button

### PR DESCRIPTION
**Pull request by annando**
This fixes the problem that resharing a reshared post changed the resharer. And also the "announce" button is added in Frio for mobile users.

### CODE REVIEW

**Review by MrPetovan — APPROVED** [review]

**Comment by annando** [comment]
@MrPetovan do you want to merge it or should I add some stuff? I already had done some more work, like announcing of Diaspora posts and renaming the functions in "Share" and "Quoted Share".

**Comment by MrPetovan** [comment]
You can add more stuff.

**Comment by annando** [comment]
@MrPetovan added

File: src/Object/Post.php [hunk 1]
```diff
@@ -346,10 +346,10 @@ public function getTemplateData(array $conv_responses, $thread_level = 1)
 			$buttons['like']    = [DI::l10n()->t("I like this \x28toggle\x29")      , DI::l10n()->t("like")];
 			$buttons['dislike'] = [DI::l10n()->t("I don't like this \x28toggle\x29"), DI::l10n()->t("dislike")];
 			if ($shareable) {
-				$buttons['share'] = [DI::l10n()->t('Share this'), DI::l10n()->t('Share')];
+				$buttons['share'] = [DI::l10n()->t('Quote and share this'), DI::l10n()->t('Quoted Share')];
```
Comment (MrPetovan) [inline src/Object/Post.php]: A "quoted share" has a specific meaning in English finance,
```suggestion
				$buttons['share'] = [DI::l10n()->t('Quote and share this'), DI::l10n()->t('Quote Share')];
```

**Review by MrPetovan — CHANGES_REQUESTED** [review]

**Review by MrPetovan — CHANGES_REQUESTED** [review]
I believe you need to update the messages.po after the last translation string change.

**Review by MrPetovan — APPROVED** [review]

**Pull request merged**